src/main/java/com/rhombus/api/examples/UserList.java

```diff
@@ -0,0 +1,146 @@
+package com.rhombus.api.examples;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.rhombus.ApiClient;
+import com.rhombus.sdk.UserWebserviceApi;
+import com.rhombus.sdk.domain.*;
+import org.apache.commons.cli.*;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.http.Header;
+import org.apache.http.message.BasicHeader;
+import org.glassfish.jersey.jackson.internal.jackson.jaxrs.json.JacksonJaxbJsonProvider;
+
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLSession;
+import javax.ws.rs.client.ClientBuilder;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class UserList
+{
+
+    private static ApiClient _apiClient;
+    private static UserWebserviceApi _userwebservice;
+
+    public static void main(String[] args) throws Exception
+    {
+        final Options options = new Options();
+        options.addRequiredOption("a", "apikey", true, "API Key");
+        options.addOption("c", "csv", true, "Name the csv file for the names");
```

Comment:
Options -c and -p should be one argument.  for example, I should just be able to specify `-c /home/users.csv` and you would just try to open the file `/home/users.csv` to write to.  If you can't open it, it is okay to print an error message and exit.